spark-publish/index.js

```diff
@@ -16,7 +16,7 @@ export const publish = async ({
       id,
       spark_version,
       zinnia_version,
-      wallet_address,
+      wallet_address as participant_address,
```

Comment:
can we also reflect this change in the schema?

Reply:
and in the data the spark client submits in the first place?

Reply:
Great ideas 👍🏻